GoogleForCreators/web-stories-wp
# Animations: Editor crashes when using negative duration values

## Context


The UI would crash when a user entered a negative value for one of the animation properties.

## Summary

 
Changes to the `sanitizeTimings` function as well as the `onChange` handler were made to reset negative values set by the user to `0`.

## Relevant Technical Choices



## To-do



## User-facing changes

Negative values can be entered, but will be reset to `0` to avoid a UI crash.

| Before                                                                                               | After                                                                                                |
|------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
| https://user-images.githubusercontent.com/4173034/168898207-12fd9203-5d0a-48db-a329-a44bbfb2d636.mov | https://user-images.githubusercontent.com/4173034/168898488-3b2fd8f6-87d6-4928-adf1-2b51b5687f86.mov |


## Testing Instructions



- For a new or existing story, add animation to a page element on a page that allows animations
- Enter a negative value into the `Duration` or `Delay` field
- Value should reset to `0` on blur or <key>Enter</key>

This PR can be tested by following these steps:

1. Clone this branch
2. Startup a local dev server
3. Follow testing instructions above


## Reviews

### Does this PR have a security-related impact?


No

### Does this PR change what data or activity we track or use?


No

### Does this PR have a legal-related impact?


No

## Checklist




---

Fixes #11439
